vtex-apps/app-store
# Change review order page redirect

#### What is the purpose of this pull request?

Change Review Order's page redirect from `/setup` to `/install`

#### What problem is this solving?

This partially addresses https://github.com/vtex/admin-apps/issues/120

#### How should this be manually tested?

Visit this [linked App Store](https://artur--extensions.myvtex.com/) and follow an app install flow. You should reach `https://{{account}}.myvtex.com/admin/apps/{{appID}}/install`

#### Screenshots or example usage
None

#### Types of changes
